Sitefinity CLI/Commands/AddResourcePackageCommand.cs

```diff
@@ -60,5 +61,44 @@ public override int OnExecute(CommandLineApplication config)
 
             return (int)ExitCode.OK;
         }
+
+        internal string SanitizedName
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(this.sanitizedName))
+                {
+                    this.sanitizedName = this.GetSanitizedName(this.Name);
+                }
+
+                return this.sanitizedName;
+            }
+        }
+
+        internal string GetSanitizedName(string name)
+        {
+            if (Char.IsDigit(name[0]))
+                name = "_" + name;
+
+            for (int i = 0; i < name.Length; i++)
+            {
+                UnicodeCategory cat = char.GetUnicodeCategory(name[i]);
+
+                if ((cat != UnicodeCategory.UppercaseLetter) && (cat != UnicodeCategory.LowercaseLetter) &&
```

Comment:
Maybe if we add the unsupported UnicodeCategories into a some kind of list and check if our is in it, it will be more readable